### PR TITLE
Mscr 119

### PR DESCRIPTION
**Pull request by ilya-wd**

### CODE REVIEW

File: build.gradle
Comment (jkesanie) [inline build.gradle:1]: Cleanup

File: src/main/java/fi/vm/yti/datamodel/api/v2/dto/ModelConstants.java
Comment (jkesanie) [inline src/main/java/fi/vm/yti/datamodel/api/v2/dto/ModelConstants.java:1]: Cleanup

File: src/main/java/fi/vm/yti/datamodel/api/v2/dto/SchemaDTO.java [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import java.util.Set;
 import java.util.UUID;
 
+
```
Comment (jkesanie) [inline src/main/java/fi/vm/yti/datamodel/api/v2/dto/SchemaDTO.java:4]: Insert nitpicking here

File: src/main/java/fi/vm/yti/datamodel/api/v2/mapper/OrganizationMapper.java
Comment (jkesanie) [inline src/main/java/fi/vm/yti/datamodel/api/v2/mapper/OrganizationMapper.java:1]: Cleanup

File: src/main/java/fi/vm/yti/datamodel/api/v2/service/impl/PostgresStorageService.java
Comment (jkesanie) [inline src/main/java/fi/vm/yti/datamodel/api/v2/service/impl/PostgresStorageService.java:1]: Clean up

File: src/main/resources/logback-spring.xml
Comment (jkesanie) [inline src/main/resources/logback-spring.xml:1]: Revert these changes. The Default logging level should be INFO.

**Review by jkesanie — CHANGES_REQUESTED** [review]

File: src/main/java/fi/vm/yti/datamodel/api/v2/dto/ModelConstants.java [hunk 2]
```diff
@@ -12,7 +12,7 @@ private ModelConstants(){
     public static final String SUOMI_FI_NAMESPACE = "http://uri.suomi.fi/datamodel/ns/";
     public static final String URN_UUID = "urn:uuid:";
     public static final String DEFAULT_LANGUAGE = "fi";
-    public static final List<String> USED_LANGUAGES = List.of("fi", "sv", "en");
```
Comment (username-u) [inline src/main/java/fi/vm/yti/datamodel/api/v2/dto/ModelConstants.java:15]: no whitespace

**Review by jkesanie — APPROVED** [review]
I added a commit to revert the logging configs back to less verbose ones. 
Otherwise ok.

**Pull request merged**